shell/layouts/default.vue

```diff
@@ -267,6 +268,11 @@ export default {
   mounted() {
     // Sync the navigation tree on fresh load
     this.$nextTick(() => this.syncNav());
+
+    this.setTabVisibilityListener(true);
```

Comment:
I'm not sure this would work but would make it cleaner, can both listen and un-listen be moved into a mixin? Would the mounted and beforeDestroy hooks in the parent both execute if there's some in the child mixin as well?

Reply:
Yep, it worked on the mounted hook, so moved it inside the mixin. Thanks!